README.md

```diff
@@ -1,15 +1,96 @@
 # Babel Polyfills
 
-A set of Babel plugins that enable injecting different polyfills with different strategies in your compiled code.
-Additionally, this reporitory contains a package that helps creating providers for any other polyfill.
-
-> ⚠️ These packages are highly experimental and not published yet. You can use GitHub's "Watch for Releases only" feature to be notified when the first version is published.
+> ⚠️ These packages are **highly experimental** and they have not been tested in production applications yet. Also, we are still working ond wiring some missing polyfills.
 
 > ℹ️ This repository implements what was initially proposed at [babel/babel#10008](https://github.com/babel/babel/issues/10008).
 
 > 💡 If you are looking for some quick setup examples, or just want to see how to migrade your config, please check [`docs/migration.md`](https://github.com/babel/babel-polyfills/blob/master/docs/migration.md).
 
-## How does it work?
+## Usage
+
+The main Babel packages only transform JavaScript _syntax_: you also need to load a polyfill, to make native _functions_ or _built-in objects_ work in older browsers. The Babel plugins implemented in this repository automatically inject the polyfills in your code, trying to only load what is really needed basd on your compilation targets and on what you are using in your code.
```

Comment:
```suggestion
The main Babel packages only transform JavaScript _syntax_: you also need to load a polyfill, to make native _functions_ (`Array.prototype.flat`) or _built-in objects_ (`Promise`) work in older browsers. The Babel plugins implemented in this repository automatically inject these polyfills in your code, while trying to only load what is really needed. It does this based on your compilation targets and on what you are using in your code.
```

Reply:
Actually would like to add a short thing about how you would normally do `<script src=` or do `import "polyfill"` at the top of your file but that assumes your targets don't change or that you are using that polyfill.

Reply:
Could be good in history section too